backend/src/db/mongoose.js

```diff
@@ -1,7 +1,7 @@
 import mongoose from 'mongoose';
 import { DATABASE_URL } from '../config';
 
-const initDb = () => mongoose.connect(DATABASE_URL, {
+const initDb = () => mongoose.connect(process.env.MONGO_URL || DATABASE_URL, {
```

Comment:
use env variables from a config file

Reply:
dist files are used to serve our nodejs backend so needed to commit these files

Reply:
MONGO_URL is not present in config file, it is generated by preset: '@shelf/jest-mongodb'